### ISSUE TITLE
Uncaught error doesn't contain message in plugin-encryption

### ISSUE THREAD

**Issue by andrewholsted** [ciscospark/spark-js-sdk#726]
There's a scenario where an error message is not being associated with the relevant error. 
<img width="461" alt="screen shot 2017-10-13 at 1 21 40 pm" src="https://user-images.githubusercontent.com/2391772/31566229-80c8a130-b01e-11e7-9a55-e1b74e3d1173.png">

The stack trace is hard to follow but it's sometime after `fetchKey`

**Comment by ianwremmel** [comment]
just look for `new KmsError()` or any of the constructors derived from kms error. Ideally, we would never construct such an error without a message. moreover, we should improve KmsError's `toString()`  to maybe include the kms request parameters and (if possible), the webex tracking id.

**Comment by ianwremmel** [comment]
I think I found a bug in the base Exception class which caused this. Hopefully should be fixed in the next few days.
